### ISSUE TITLE
Some issues encountered

### ISSUE THREAD

**Issue by himouto** [Sirosky/go-reader#4]
- can't import to library if image file names has spaces on them, has to rename them in order to import
- can't import to library if the folder name that contains the images has spaces too
- if manually import files with spaces on their file names to the library folder, it won't load them
- if the images of a chapter has file names "1".jpg to "20".jpg it will load them at a wrong order: 1, 10, 11, 12.. 19, 20, 2, 3..9 instead of 1, 2, 3.. 20. only fix for this is to add a "0" to the single digit file names (01, 02, 03..)

**Comment by Sirosky** [comment]
Thanks for flagging these issues-- I've been super busy so unlikely I'll be able to get around to debugging/ fixing them =/

I ran a quick test on the third point, and spaces in the file name seems to work for me.  With regards to the fourth point, yeah that does indeed seem to be a problem. I'll probably have to write a parser for file names at some point, but not entirely sure how to solve it at the moment. 

In any case, I will leave the issue open. Hopefully I'll be able to get around to them at some point.

**Comment by himouto** [comment]
3rd issue could be misunderstood.. meant if i directly import the image files (that has spaces on their filenames) to the library folder through windows explorer (not using the reader's import function) it doesn't load by the reader but if i rename them and remove the spaces from their filenames they load properly

**Comment by Sirosky** [comment]
Yeah I added spaces into image files that I had already in my library and they seemed to load fine. Can you give an example of a file name that didn't work?

**Comment by himouto** [comment]
"Page 001.png" - not sure if it was the space or the file format. i just pasted the entire chapter folder directly to the library folder via explorer because it won't import through the reader and when i try to load them it just shows blank pages like this https://streamable.com/vb8z6l

**Comment by Sirosky** [comment]
Thanks for the example. Very odd because I tried that file name and it loaded fine for me. Unfortunately, I don't think there are any straightforward solutions to any of the issues you flagged and due to IRL commitments, I won't have time to debug/look into these for a while.

If you're up for it, the source is available as well.

**Comment by himouto** [comment]
Okay dude, just wanted to report these in case other's tried to fork and improve the application.. this is the only local viewer i found for windows with infinite scrolling, just unfortunate it has issues like these..

**Comment by Sirosky** [comment]
Yup thanks for reporting these issues. Unfortunately it's part of the problem with using a game engine to make an app like this. I'll keep this issue open until I have time (someday) to look into them or if someone else can.

**Comment by MaverickTesla** [comment]
The only thing that can also be improvised, in the import menu and path modification dialogue box....!! Although the software is Super Easy with flawless experience while reading the CBR files, !! Thank you for contributing your work !! 💖💖💖
